ledger/participant-integration-api/src/main/scala/platform/store/backend/StorageBackend.scala

```diff
@@ -47,6 +74,18 @@ trait StorageBackend[DB_BATCH] {
     * @return the LedgerEnd, which should be the basis for further indexing.
     */
   def initialize(connection: Connection): StorageBackend.LedgerEnd
+}
+
+// TODO append-only: consolidate parameters table facing functions
+trait ParameterStorageBackend {
+
+  /** This method is used to update the parameters table: setting the new observable ledger-end, and other parameters.
+    * No significant CPU load, mostly blocking JDBC communication with the database backend.
+    *
+    * @param connection to be used when updating the parameters table
+    * @param params     the parameters
+    */
+  def updateParams(connection: Connection, params: StorageBackend.Params): Unit
```

Comment:
This method's signature doesn't follow the curried pattern with `def someSetter(theSetteee: Bla)(connection: Connection)`

Reply:
I thought we will do this as part of the parameters consolidation (which I have hoped to do before sooner than later). Since you mentioned, and since the naming should be something final for the interface, I am fixing it now